packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -104,7 +104,9 @@ export class DynamoDBModelTransformer extends Transformer {
                 "^" + def.name.value + "Table",
                 // All resolvers except the search resolver.
                 "^[^S].*" + def.name.value + "Resolver",
-                "^" + def.name.value + ".+Resolver"
+                "^" + def.name.value + ".+Resolver",
+                def.name.value + "Table",
```

Comment:
This should be handled by the `"^" + def.name.value + "Table",` above. I believe you should be able to remove one of these and prefer the more specific rule as long as it does not interact with the migration issue.

Reply:
@mikeparisstuff you are right, but, the one that solves the issue with the migration is `def.name.value + "Table",`. I will make some tests to make sure that removing `"^" + def.name.value + "Table",` doesn't break anything that is within my reach.

Reply:
@mikeparisstuff I've removed `"^" + def.name.value + "Table",` and confirmed everything works before and after migration from a previous version of the CLI.